docs/blog/2024-07-25-nx-19-5-update.md

```diff
@@ -0,0 +1,117 @@
+
+However, previously when using this approach, there were some limitations regarding parallelism. Most end-to-end tests cannot run on the same machine at the same time due to shared ports, conflicting database states, etc.
+
+The solution in the past was to add a `--parallel=1` option, which was a bit clunky. For example:
+
+```terminal
+> nx run-many --targets=lint,test,e2e,build --parallel=1
+```
+
+This command above is used to run all lint, test, e2e, and build tasks present in your monorepo. The `--parallel=1` option is needed in order for your e2e tasks to work properly (so as to avoid the problems of running multiple end-to-end tests on the same machine at one time).
+
+Howeve in this scenario we'd also be limiting the parallelism of all our other tasks - which are fine to run in parallel on the same machine.
+
+To address this, all tasks now support a `parallelism` property. By setting this property to `false`, you can now inform the Nx task runner to not attempt to run a certain task in parallel. For example setting this in your `targetDefaults` will mark all atomized e2e tasks in to not run in parallel:
+
+```json
+{
+  // ...
+  "targets": {
+    "e2e-**/*": {
+      // ...
+      "parallelism": false
+    }
+  }
+}
+```
+
+This way we can simply run the command:
+
+```terminal
+> nx run-many --targets=lint,test,e2e,build
+```
+
+And in CI, Nx Agents will allow all tasks to run in parallel on the same machine, except for the atomized end-to-end tasks, which will only run in isolation.
+
+## Gradle: Tests Can Be Atomized
+
+Our new `@nx/gradle` plugin now supports Test Atomization out of the box.
+
+[add image of ocean repo from nx-api]
+
+This means that as you add tests in your gradle projects, we'll automatically create a new tasks for each test class, allowing you to distribute the execution of these tasks in your CI pipeline.
+
+## WASM (and StackBlitz!!) Support
+
+Nx now has support for WASM - or WebAssembly. This means that you can create a StackBlitz with a whole Nx Workspace inside of it and run all Nx capabilities from their embedded terminal.
+
+[add gif of nx running in stackblitz]
+
+We're excited for this as it opens up many interesting use-cases, including easier ways of sharing examples, better opportunities for reproduction of issues or bugs, and potential for us to use embedded examples in our documentation in the future.
+
+## Support for React 19 (rc) and Angular 18.1
+
+Nx 19.5 adds support for the React 19 release candidate and updated our Angular package to support the latest Angular minor version: 18.1.
+
+## Monorepo World Conference Speakers to Be Announced Soon!!
+
+[![Monorepo World](/blog/images/2024-07-25/monorepo-world.jpg)](https://monorepo.world)
+
+The Monorepo World conference is coming up soon on October 7, 2024. Be sure to stay tuned as we'll be announcing speakers soon!
+
+## Learn more
```

Comment:
We should have this section on all release blog posts. We didn't have it on the last one either, but we def should:

https://nx-dev-git-nx-19-5-update-nrwl.vercel.app/blog/nx-17-2-release#automatically-update-nx